src/Javis.jl

```diff
@@ -752,4 +751,7 @@ export setup_stream, cancel_stream
 # scales
 export scale_linear, @scale_layer
 
+export morph
```

Comment:
can you remove these three export lines? Seems like they are from your other branches?